src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -70,15 +69,15 @@ def create(self, params):
         domain_id = params['domain_id']
         project_id = params.get('project_id')
         secret_project_id = self.transaction.get_meta('secret.project_id')
+        provider = params.get('provider', self.transaction.get_meta('secret.provider'))
```

Comment:
Move the logic to get the provider into a function.

Reply:
have reflected thank you.